packages/features/bookings/lib/getBookingFields.ts

```diff
@@ -0,0 +1,245 @@
+import { EventTypeCustomInput, EventType } from "@prisma/client";
+import { z } from "zod";
+
+import slugify from "@calcom/lib/slugify";
+import {
+  BookingFieldType,
+  customInputSchema,
+  eventTypeBookingFields,
+  EventTypeMetaDataSchema,
+} from "@calcom/prisma/zod-utils";
+
+type Fields = z.infer<typeof eventTypeBookingFields>;
+
+const EventTypeCustomInputType = {
+  TEXT: "TEXT",
+  TEXTLONG: "TEXTLONG",
+  NUMBER: "NUMBER",
+  BOOL: "BOOL",
+  RADIO: "RADIO",
+  PHONE: "PHONE",
+};
```

Comment:
I believe this would provide better TS support when using this 
```suggestion
const EventTypeCustomInputType = {
  TEXT: "TEXT",
  TEXTLONG: "TEXTLONG",
  NUMBER: "NUMBER",
  BOOL: "BOOL",
  RADIO: "RADIO",
  PHONE: "PHONE",
} as const;
```